jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc

```diff
@@ -32,7 +32,7 @@
 - [[[bib8,8]]]  Jakarta™ Servlet Specification, Version 4.0. Available at:
                https://jakarta.ee/specifications/servlet/4.0
 
-- [[[bib9,9]]]  Jakarta XML Web Services, Version 3.0
+- [[[bib9,9]]]  Jakarta™ XML Web Services, Version 3.0
```

Comment:
According to this page https://jakarta.ee/ it has to be ® instead of ™, as Jakarta is a *registered* trademark, actually.

Reply:
I added ™, because we already used it for some other entries in the bibliography section and because of the note in #804. Not sure which would we should use TBH.

Reply:
According to [WIPO](https://www3.wipo.int/branddb/en/) Jakarta is a *registered* trademark in the US and other countries, so it must bear the ® to clearly indiciate this.

Reply:
I chatted with @kwsutter about this.  He pointed me to this page: https://www.eclipse.org/legal/logo_guidelines.php

It says that we can use either one (® or ™).  Kevin also mentioned that we only need the trademark symbol  on the first use for a document or section. 

I don't have a strong preference either way on the ® vs ™ question.  One the one hand, it might be nice to be consistent with the jakarta.ee page - on the other hand, if it is already ™ elsewhere in the spec, I don't see enough justification to change it everywhere.  I guess my preference is for consistency in general - we shouldn't use ™ in some parts of the spec and ® in others.

Thanks @chkal!

Reply:
Andy, there is a misunderstanding. The text says "the **appropriate** ™ or ® symbol". "Appropriate" does not imply we have a free choice. From a legal aspect. ™ *must* be used for *trademarks in general*, while ® *must* be used for *registered* trademarks. So in our case, it *must* be ®.

Reply:
I'm fine with using ® instead of ™, especially for being consistent with the Jakarta website. I just update the pull request.